libs/parallelism/algorithms/include/hpx/parallel/algorithms/transform.hpp

```diff
@@ -787,7 +787,7 @@ namespace hpx { namespace traits {
     struct get_function_annotation<
         parallel::v1::detail::transform_iteration<ExPolicy, F, Proj>>
     {
-        static char const* call(
+        static constexpr char const* call(
```

Comment:
Do we know by the way if this actually has an effect on inlining or something like that? I doubt that we ever actually use the string in a constexpr context, or am I missing some other reason to make these `constexpr`?

Reply:
I was hoping that under certain circumstances the annotation for a function could be evaluated at compile time, for instance:
```
void foo();
void bar()
{
    char const* annotation = hpx::traits::get_function_annotation<&foo>();
}
```
But I have not checked whether this is actually happening.